src/Startup.cs

```diff
@@ -51,13 +58,14 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             app.UseHttpsRedirection();
             app.UseStaticFiles();
             app.UseCookiePolicy();
-
-            app.UseMvc(routes =>
+            // Config UseSwaggerUI.

```

Comment:
Same here. The `app.UseSwagger()` statement is telling the app we will use Swagger, and the `app.UseSwaggerUI(...)` is setting up the UI itself to work with the API endpoint